### PR TITLE
CASSANDRA-16618: Fix broken IntelliJ configuration

### PR DESCRIPTION
**Pull request by jacek-lewandowski**

### CODE REVIEW

**Review by jtgrabowski — APPROVED** [review]
works for me

File: ide/idea-iml-file.xml [hunk 1]
```diff
@@ -49,25 +49,27 @@
         <orderEntry type="module-library">
             <library>
                 <CLASSES>
-                    <root url="file://$MODULE_DIR$/lib" />
+                    <root url="file://$MODULE_DIR$/build/lib/jars" />
                 </CLASSES>
```
Comment (michaelsembwever) [inline ide/idea-iml-file.xml:53]: do we need both `<CLASSES>` sections? (only jar files are found in `build/lib/jars` and `build/test/lib/jars`)

Comment (jacek-lewandowski) [reply ide/idea-iml-file.xml:53]: perhaps we do not, I've just taken the file generated by IntelliJ and those entries were there. I'll try if we can safely remove them

Comment (jacek-lewandowski) [reply ide/idea-iml-file.xml:53]: ok, unfortunately we cannot remove `CLASSES` entries, IntelliJ would recognise it as an empty library :(

**Comment by azotcsit** [comment]
Looks like this PR has been merged and can be closed now.

**Pull request closed without merging**